subworkflows/local/vcf_concatenate_germline/main.nf

```diff
@@ -13,10 +14,14 @@ workflow CONCATENATE_GERMLINE_VCFS {
 
     take:
     vcfs
+    fasta
 
     main:
     versions = Channel.empty()
 
+    // Normalize vcf-files (added by jcdelmas 240415)
```

Comment:
we have a section in the readme collecting all contributors for more visibility. Just add yourself there 🙂 : https://github.com/nf-core/sarek/#credits